docs/user-guide/kubernetes.md

```diff
@@ -67,6 +67,64 @@ To deploy Træfɪk to your cluster start by submitting the deployment to the clu
 ```sh
 kubectl apply -f examples/k8s/traefik.yaml
 ```
+### Role Based Access Control configuration (optional)
+
+Kubernetes introduces [Role Based Access Control (RBAC)](https://kubernetes.io/docs/admin/authorization/) in 1.6+ to allow fine-grained control
+of Kubernetes resources and api.
+
+If your cluster is configured with RBAC, you need to authorize Træfɪk to use
+kubernetes API using ClusterRole, ServiceAccount and ClusterRoleBinding resources:
+
+```yaml
```

Comment:
I'm okay with keeping the inline spec, especially since you and @errm are in favor of it. Happy to let myself get overruled by the majority. :-)

As soon as you address [the other point](https://github.com/containous/traefik/pull/1404#discussion_r111105543), we can merge.